python-avd/pyavd/templater.py

```diff
@@ -112,7 +116,7 @@ def import_filters_and_tests(self) -> None:
     def render_template_from_file(self, template_file: str, template_vars: dict) -> str:
         return self.environment.get_template(template_file).render(template_vars)
 
-    def compile_templates_in_paths(self, precompiled_templates_path: str, searchpaths: list[str]) -> None:
+    def compile_templates_in_paths(self, precompiled_templates_path: str, searchpaths: list[str], log_level: int = logging.DEBUG) -> None:
```

Comment:
Why have it as debug as default? I mean we have the loggic level for pyavd we could rely on?

Reply:
we don't but this was never used except in compile_schemas script which is quite verbose